pkgs/development/tools/darklua/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "darklua";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "seaofvoices";
+    repo = "darklua";
+    rev = "v${version}";
+    hash = "sha256-ABzhtAbWv2oM4VqxRUWC+xh2fwUw1s2iU1IWb5EEhiE=";
+  };
+
+  cargoHash = "sha256-36YN/7aI69xsnwGz8oQG5RZu8XjQ9vwQtRNMWQ7pT0Q=";
+
```

this makes it work on aarch64-linux

```suggestion

  # error: linker `aarch64-linux-gnu-gcc` not found
  postPatch = ''
    rm .cargo/config.toml
  '';

```

---

Fixed in latest push.